promql/engine.go

```diff
@@ -750,6 +752,7 @@ func (ev *evaluator) vectorOr(lhs, rhs vector, matching *VectorMatching) vector
 	if matching.Card != CardManyToMany {
 		panic("logical operations must always be many-to-many matching")
 	}
+	// If no matching labels are specified, match by all labels.
```

Comment:
I removed the comments, the behavior of the signatur function should document its behavior, such comments will just get out of sync.